index.md

```diff
@@ -62,16 +62,13 @@ If you're new to the project you should start with our [getting started](docs/38
 
 ## Community
 
-To get in touch with us, you can try to reach us on **IRC** at **\#389** on **irc.libera.chat** or on one of our [mailing lists](docs/389ds/mailing-lists.html).
+To get in touch with us, you can reach us on slack at #team-ldap or on one of our [mailing lists](docs/389ds/mailing-lists.html).
```

Comment:
I am unsure slack channel would be available for the community

Reply:
Right, this is internal instance. We still have presence on libera.chat

Reply:
Gulp, my bad. I thought the opposite for both slack and libera.chat